client/app/account/components/DeleteAccount.tsx

```diff
@@ -1,45 +1,61 @@
+import React, { useState } from 'react'
 import { FiTrash } from 'react-icons/fi'
-import {
-  AlertDialog,
-  AlertDialogAction,
-  AlertDialogCancel,
-  AlertDialogContent,
-  AlertDialogDescription,
-  AlertDialogFooter,
-  AlertDialogHeader,
-  AlertDialogTitle,
-  AlertDialogTrigger,
-  Button,
-} from '@/_components/ui'
+import { Button } from '@/_components/ui'
+import { HouseType, UserType } from '@/_types'
 import { useDeleteAccount } from '../hooks'
+import { DeleteAccountModal, AdminWarningModal } from '.'
 
-const DeleteAccount = () => {
+type DeleteAccountProps = {
+  user: UserType
+  userHouse: HouseType | undefined
```

Comment:
You can delete undefined here!

Reply:
Same here too. This is for a case where a user exists but doesn't yet have a house.

Reply:
Fixed it
2daa9364744489095cfa7a294906c1f4f44d6f48